zipline/algorithm.py

```diff
@@ -1299,7 +1310,15 @@ def symbols(self, *args):
         --------
         :func:`zipline.api.set_symbol_lookup_date`
         """
-        return [self.symbol(identifier) for identifier in args]
+        country_code = kwargs.pop('country_code', None)
```

Comment:
My kingdom for keyword-only args...

Reply:
Actually, is there a reason to prefer this over forwarding `**kwargs` to `self.symbol`? I guess it's slightly nicer to have the traceback refer to `symbols` instead of `symbol`?

Reply:
oh, I guess I could do that